libshortfin/src/shortfin/local/scheduler.cc

```diff
@@ -36,6 +39,29 @@ void Account::active_deps_extend(iree_hal_semaphore_list_t sem_list) {
   }
 }
 
+CompletionEvent Account::OnSync() {
+  bool hack_has_working_wait_source = false;
+  if (hack_has_working_wait_source) {
+    return CompletionEvent(sem_, idle_timepoint_);
+  } else {
+    // TODO: Burn this path with fire! No attempt has been made to make this
```

Comment:
![200w 1](https://github.com/user-attachments/assets/c4347d55-7146-4ec2-b9a4-765ab5726901)

Reply:
lol, 'tis ok. Forced me to wire in a blocking executor, which is needed for other things.